readme.md

```diff
@@ -274,6 +276,35 @@ function usePick(): (...routes: Array<Route>) => Route
 
 Returns a function that lets you choose a route from an array of routes, or `null` if nothing matched. The selected route is found by traversing the parents of the picked route (`useLocationMatch`).
 
+---
+### `useIsCurrent`
+
+```js
+function useIsCurrent(route, params = {}): boolean
+```
+
+Returns a boolean indicating whether the route and params match the route and params of the current location. Note that if you don't want to match the params you can use the following:
```

Comment:
Het duurde voor mij even voor dat ik snapte waar dit over ging, ik zou zelf even wat extra voorbeelden toevoegen waar bij je laat zien waarom je dit wel of niet zo willen. Zeker het verschil tussen de twee hooks wordt voor mij niet helemaal duidelijk.

Reply:
"Je kijkt dus eigenlijk niet of dat de routemapRoute.toString() matched of zo, maar meer of dat het route object en de params objecten het zelfde zijn."